_config.yml

```diff
@@ -39,41 +39,45 @@ defaults:
 
 authors:
   etagwerker:
-    display_name: Ernesto Tagwerker
+    display_name: "Ernesto Tagwerker"
     gravatar: 9299203e7e9b9ae35e39eb66f3155f15
     email: ernesto@ombulabs.com
     twitter: etagwerker
     github: etagwerker
+    bio: "Software engineer and open source maintainer. He is a lean practitioner and entrepreneur working on software since 2004."
   mauro-oto:
-    display_name: Mauro Otonelli
+    display_name: "Mauro Otonelli"
     gravatar: 350875eb8fce8553e12040c5a1f5d5d0
     email: mauro@ombulabs.com
     twitter: mauro_oto
     github: mauro-oto
   luciano:
-    display_name: Luciano Becerra
+    display_name: "Luciano Becerra"
     gravatar: 9219f1f8634fafcbed16e72a0bb5a414
     email: luciano@ombulabs.com
     twitter: lubc32
     github: lubc
+    bio: "Passionate about programming, entrepreneurship, startups, and all things related to technology."
   emily:
-    display_name: Emily Cohen
+    display_name: "Emily Cohen"
     gravatar: 1abbc4d93954d2df607cc3ec7fa9f418
     email: emily@ombulabs.com
     twitter:
     github: ercohen14
   cleiviane:
-    display_name: Cleiviane Costa
+    display_name: "Cleiviane Costa"
     gravatar: db42db5ca02074d39438c612433f2209
     email: cleiviane@ombulabs.com
     twitter:
     github: cleivianecosta
+    bio: "A Brazilian Software Engineer passionate about code and technology. Rubik's Cube Fan, LoL player and coffee lover."
```

Comment:
@etagwerker I need to change my bio. Not a LoL player anymore haha

What do you think about:

"A Brazilian Software Engineer passionate about code, entrepreneurship and technology. She likes to think out of the box and bring new ideas for the team."

Reply:
@cleicar Sure, no problem. I just updated it. 😄 

You might want to submit a PR for https://github.com/ombulabs/fastruby.io (I originally grabbed your bio from that page) 😛